PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -203,94 +203,113 @@ fi
-									a positive phase encoding direction (RL in
-									HCP data), set to "NONE" if not using Spin
-									Echo Field Maps
-  --seechospacing=<seconds>         Effective Echo Spacing of Spin Echo Field Map,
-									(in seconds) or "NONE" if not used
-  --seunwarpdir={x,y,NONE}          Phase encoding direction (according to the *voxel* axes)
-             or={i,j,NONE}          of the spin echo field map. 
-                                    (Only applies when using a spin echo field map.)
-  --t1samplespacing=<seconds>       T1 image sample spacing, "NONE" if not used
-  --t2samplespacing=<seconds>       T2 image sample spacing, "NONE" if not used
-  --unwarpdir={x,y,z,x-,y-,z-}      Readout direction of the T1w and T2w images (according to the *voxel axes)
-           or={i,j,k,i-,j-,k-}      (Used with either a gradient echo field map 
-									or a spin echo field map)
-  --gdcoeffs=<file path>            File containing gradient distortion
-									coefficients, Set to "NONE" to turn off
-  --avgrdcmethod=<avgrdcmethod>     Averaging and readout distortion correction
-									method. See below for supported values.
-
-	  "${NONE_METHOD_OPT}"
-		 average any repeats with no readout distortion correction
-
-	  "${FIELDMAP_METHOD_OPT}"
-		 equivalent to "${SIEMENS_METHOD_OPT}" (see below)
-		 SiemensFieldMap is preferred. This option value is maintained for
-		 backward compatibility.
-
-	  "${SPIN_ECHO_METHOD_OPT}"
-		 average any repeats and use Spin Echo Field Maps for readout
-		 distortion correction
-
-	  "${GENERAL_ELECTRIC_METHOD_OPT}"
-		 average any repeats and use General Electric specific Gradient
-		 Echo Field Maps for readout distortion correction
-
-	  "${SIEMENS_METHOD_OPT}"
-		 average any repeats and use Siemens specific Gradient Echo
-		 Field Maps for readout distortion correction
-
-  --topupconfig=<file path>      Configuration file for topup or "NONE" if not
-								 used
-  --bfsigma=<value>              Bias Field Smoothing Sigma (optional)
+  --path=<path>                       Path to study data folder (required)
+                                      Used with --subject input to create full path to root
+                                      directory for all outputs generated as path/subject
+  --subject=<subject>                 Subject ID (required)
+                                      Used with --path input to create full path to root
+                                      directory for all outputs generated as path/subject
+  --t1=<T1w images>                   An @ symbol separated list of full paths to T1-weighted
+                                      (T1w) structural images for the subject (required)
+  --t2=<T2w images>                   An @ symbol separated list of full paths to T2-weighted
+                                      (T2w) structural images for the subject (required for 
+                                      hcp-style data, can be NONE for legacy-style data)
```

Comment:
I suggest we say: "... can be NONE for legacy-style data, see --mpp-mode option)".

Reply:
Change.